src/main/webapp/gallery.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Gallery</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
```

Comment:
You don't need to include script.js in this page. It only affects pages with 'greeting-container' element.

Reply:
That is. My commit add my entire directory, so there is some work that has not been completed